src/test_mailroom_madness.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""File tests a mailroom helper function.
+
+Attributes:
+    TEST_INTS (TYPE): Description
+    TEST_NAMES (TYPE): Description
+"""
+import pytest
+
+TEST_INTS = [
+    (u'13', 13),
+    (u'49', 49),
+    (u'35', 35)
+]
+TEST_DONATION_VALIDATION = [
+    (u'14', True),
+    (u'348', True),
+    (u'-28', False),
+    (u'taco', False),
+    (u'', False)
+]
+TEST_NAMES = [
+    u'Jennifer Lange',
+    u'Foo Bar',
+    u'Hello World'
+]
+TEST_MAX_SIZE = [
+    ([u'string', 6, 34, u'taco'], 6),
+    ([873, u'Bob Saget', 387263743], 9)
+]
+
+
+def test_prompt_donor_name():
+    """Function tests prompt_donor_name with test data."""
+    import mailroom_madness
+    mailroom_madness.input = lambda _: u'Jennifer White'
```

Comment:
This is a form of "mocking".  Which works.  But I'd much rather have you solve the problem of isolating your input sources from the rest of your running code.  It's a more useful architectural lesson than learning how to mock through monkey-patching.